ifacemaker.go

```diff
@@ -5,45 +5,68 @@ import (
 	"io/ioutil"
 	"log"
 
+	"github.com/kazu/ifacemaker/maker"
 	"github.com/mkideal/cli"
-	"github.com/vburenin/ifacemaker/maker"
 )
 
 type cmdlineArgs struct {
 	cli.Helper
-	Files      []string `cli:"*f,file" usage:"Go source file to read"`
-	StructType string   `cli:"*s,struct" usage:"Generate an interface for this structure name"`
-	IfaceName  string   `cli:"*i,iface" usage:"Name of the generated interface"`
-	PkgName    string   `cli:"*p,pkg" usage:"Package name for the generated interface"`
-	CopyDocs   bool     `cli:"d,doc" usage:"Copy docs from methods" dft:"true"`
-	Output     string   `cli:"o,output" usage:"Output file name. If not provided, result will be printed to stdout."`
+	Files          []string `cli:"*f,file" usage:"Go source file to read"`
+	StructType     string   `cli:"*s,struct" usage:"Generate an interface for this structure name"`
+	IfaceName      string   `cli:"*i,iface" usage:"Name of the generated interface"`
+	PkgName        string   `cli:"*p,pkg" usage:"Package name for the generated interface"`
+	ExcludeMethods []string `cli:"e,exclude" usage:"Method name for not generation " `
+	CopyDocs       bool     `cli:"d,doc" usage:"Copy docs from methods" dft:"true"`
+	Output         string   `cli:"o,output" usage:"Output file name. If not provided, result will be printed to stdout."`
 }
 
 func run(args *cmdlineArgs) {
 	allMethods := []string{}
 	allImports := []string{}
 	mset := make(map[string]struct{})
 	iset := make(map[string]struct{})
+	parseds := make(map[string]*maker.Parsed)
+	fmt.Printf("files=%s\n", args.Files)
```

Comment:
is a debug print output? it should not be here.

Reply:
sorry. i will remove